platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/LoggingSystem.java

```diff
@@ -52,6 +64,11 @@ public class LoggingSystem implements LogEventConsumer {
      */
     public static final String ROOT_LOGGER_NAME = "";
 
+    /**
+     * The Configuration object
+     */
+    private final Configuration configuration;
```

Comment:
Should we add notNull here too, so is consistent and forces us to check in each use?